workshop/Lab3/README.md

```diff
@@ -100,7 +100,7 @@ try to add, or remove, pods from the system to match your request. To can
 make these modifications by using the following command:
 
    ```console
-   $ kubectl edit deployment guestbook
+   $ kubectl edit deployment guestbook-v1
```

Comment:
why would the deployment be called this?

Reply:
I see, guestbook has been updated. We need a better way to keep this in sync if we're going to have separate repos. Checking out a specific revision/branch or something.

Reply:
Yeah it had to do with the tag. Is there something you want me to change in this PR?